edward/util.py

```diff
@@ -434,12 +434,16 @@ def rbf(x, y=0.0, sigma=1.0, l=1.0):
 
 def set_seed(x):
     """Set seed for both NumPy and TensorFlow.
-
     Parameters
     ----------
     x : int, float
         seed
     """
+    if len(tf.get_default_graph()._nodes_by_id.keys()) > 0:
+        from nose.tools import set_trace; set_trace()
```

Comment:
can you remove this import line, and also undelete the line that was deleted above?